content/millennium/dstu2/scheduling/appointment.md

```diff
@@ -197,6 +197,71 @@ _Implementation Notes_
 
 The `ETag` response header indicates the current `If-Match` version to use on subsequent updates.
 
+## Update
+
+Update an Appointment.
+
+    PUT /Appointment/:id
+
+_Implementation Notes_
+
+* The only supported change is to update the [Appointment.status] to either `arrived` or `cancelled`.
+* `Appointment.participant` must have at least one participant.
+* `Appointment.participant.status` must be `accepted` for each participant
```

Comment:
Probably an "all other fields will be ignored" or similar statement? We don't check for changes, correct?